flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamGraph.java

```diff
@@ -637,19 +638,24 @@ public StreamNode getStreamNode(Integer vertexID) {
 		return streamNodes.keySet();
 	}
 
+	@VisibleForTesting
 	public List<StreamEdge> getStreamEdges(int sourceId, int targetId) {
-
 		List<StreamEdge> result = new ArrayList<>();
 		for (StreamEdge edge : getStreamNode(sourceId).getOutEdges()) {
 			if (edge.getTargetId() == targetId) {
 				result.add(edge);
 			}
 		}
+		return result;
+	}
 
+	@VisibleForTesting
+	@Deprecated
+	public List<StreamEdge> getStreamEdgesOrThrow(int sourceId, int targetId) {
```

Comment:
Like I said, we should move the old behavior into the tests. If we don't do it now, it's probably okay.

Reply:
Yes, we should. As I wrote, the old method shouldn't exists. At least here I renamed it so it is not so confusing :/

But I didn't want to spend even more time cleaning up unrelated old code (because I was cleaning up/refactoring already so much).